content/en/docs/Installation/installation-guide/creating-updating-kiali-cr.md

```diff
@@ -15,13 +15,17 @@ $ kubectl annotate kiali my-kiali -n istio-system --overwrite kiali.io/reconcile
 {{% /alert %}}
 
 The Operator provides comprehensive defaults for all properties of the Kiali
-CR. Hence, the minimal Kiali CR does not have a `spec`:
+CR. You do, however, need to specify the version to install. Normally, you
+can just set this to `default` which tells the Kiali Operator to install
+the Kiali Server whose version is the same as that of the operator itself:
```

Comment:
```suggestion
CR. You do, however, need to specify the version to install. Normally, you
can just set this to `default` which tells the Kiali Operator to install the
Kiali Server whose version is the same as that of the operator itself. But, to
update the Kiali Operator while maintaining a fixed Kiali Server, you must
set the specific Kiali Server version.
```

You can re-word it, or maybe show the version format, but I think it would be good to be a little more elaborate here.

Reply:
re-worded